jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,471 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosaic_id = basemaps['mosaics'][0]['id']\n",
+    "mosaic_box = basemaps['mosaics'][0]['bbox']\n",
+    "mosaic_box_str = ','.join(map(str, mosaic_box))\n",
+    "\n",
+    "quad_params = {\n",
+    "    'mosaic_id': mosaic_id,\n",
+    "    'bbox': mosaic_box_str,\n",
+    "}\n",
+    "\n",
+    "quads_url = \"{}/{}/quads\".format(BASEMAP_API_URL, mosaic_id)\n",
+    "quad_service_response = requests.get(\n",
+    "    url=quads_url, params=quad_params, auth=auth, headers=headers)\n",
+    "quads = quad_service_response.json()\n",
+    "items = quads['items']\n",
+    "\n",
+    "quad_ids = [i['id'] for i in items]"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an order packet\n",
+    "---\n",
+    "Package up the details of your order in a [JSON object](https://developers.planet.com/apis/orders/basemaps/#example-order-query-json-block) and make a POST request, passing in the Orders URL, your JSON, your API key, and the content-type header. We are [delivering](https://developers.planet.com/apis/orders/delivery/) this order to a Google Cloud Storage bucket. In this example, we are using the tools [bandmath](https://developers.planet.com/apis/orders/tools/#band-math) and [reproject](https://developers.planet.com/apis/orders/tools/#reproject)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Reproject\n",
+    "\n",
+    "Reprojecting a basemap is necessary when the original map projection does not accurately represent the Earth's surface in a specific region or area. This can happen when the imagery for the basemap comes from a place far from the equator. In the example shown in this notebook, there is not a significant change in circumference of the earth from the equator, but it is always best to check using the code below. \n",
+    "\n",
+    "First, find the latitude of the area of interest. Then, find the radius of the Earth in kilometers. We recommend using a website like [this](https://rechneronline.de/earth-radius/) to find that number. Simply change the latitude and earth radius parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def calculate_resolution_at_latitude(latitude, earth_radius):\n",
+    "    '''Calculate the resolution of a basemap at a given latitude\n",
+    "\n",
+    "    Parameters: latitude (float): latitude at location of interest\n",
+    "                native_resolution (float): native resolution of basemap\n",
+    "    Return (float): resolution of basemap at given latitude\n",
+    "    '''\n",
+    "    earth_radius = 6370.1 # km \n",
+    "    circumference_of_earth_at_location = 2.0 * math.pi * (earth_radius * 1000) # meters\n",
```

Comment:
Here's what I'd recommend doing. 

1. swapping `earth_radius` to `native_resolution` as the second input
2. setting `native_resolution=3` in the function, so that if someone doesn't input a resolution, they can fall back on 3m.

```python
def calculate_resolution_at_latitude(latitude, native_resolution=3):
    '''Calculate the resolution of a basemap at a given latitude

    Parameters: latitude (float): latitude at location of interest
                native_resolution (float): native resolution of basemap
    Return (float): resolution of basemap at given latitude
    '''
    earth_radius = 6370.1  # km
    circumference_of_earth_at_location = 2.0 * math.pi * (earth_radius * 1000
                                                          )  # meters
    meters_per_degree_at_location = math.cos(math.radians(latitude)) * circumference_of_earth_at_location / 360.0
    degree_resolution_at_location = native_resolution / meters_per_degree_at_location
    return degree_resolution_at_location
```

Reply:
A few other things:

- `circumference_of_earth_at_location`: you're just computing the circumference of the Earth, not using latitude at all. So, I'd recommend changing this variable name to `circumference_of_earth`
- `degree_resolution_at_location`: this variable name is confusing to me. We often assume the resolution is in degrees or radians. I'd just write in the docstring that the resolution returned is in degrees.
- In fact, write the units for everything in the docstring (latitude, native_res, res_at_location)

Reply:
Ah, I realize that you probably do mean to have `earth_radius` in as an input. So,

- remove the hard-coded radius
- choose whether or not to have native res as an input (I don't think it matters too much, people will almost certainly choose 3)
- update your docstring with your actual inputs and their units